structure_tests/tests/cloudbuild.yaml.in

```diff
@@ -0,0 +1,5 @@
+steps:
+        - name: gcr.io/gcp-runtimes/structure-test-test:${TEST_TAG}
+          args: [
+          '--image', 'gcr.io/google-appengine/debian8:latest',
```

Comment:
shouldn't we run these tests against the new structure_test image itself? e.g. use `gcr.io/gcp-runtimes/structure_test` as the cloudbuild step, and `gcr.io/gcp-runtimes/structure-test-test:${TEST_TAG}` as the image under test